kwok/charts/crds/karpenter.sh_nodeclaims.yaml

```diff
@@ -816,14 +816,3 @@ spec:
       storage: false
       subresources:
         status: {}
-  conversion:
```

Comment:
We need to drop the automation here: https://github.com/kubernetes-sigs/karpenter/blob/main/hack/kwok/conversion.sh

Reply:
Yes right, dropped.